.github/workflows/build-and-test.yaml

```diff
@@ -5,16 +5,17 @@ on:
     inputs:
       scope:
         type: choice
-        description: 'Scope the workflow should run with. Allowed values are: all, build-only, test-only'
+        description: 'Scope the workflow should run with. Allowed values are: all, build-only, integration-test'
```

Comment:
From what I can see below, `integration-test` is actually `integration-test-only`, right? Could we maybe change its name to make it reflect that?

Reply:
Sure